draft-vanbrouwershaven-acme-auto-discovery.md

```diff
@@ -50,6 +50,7 @@ normative:
   RFC8659:
 
 informative:
+  RFC3986:
```

Comment:
Should this be deleted as it's not referenced?

Reply:
Done. Thanks.